src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -355,23 +350,6 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
             });
     }
 
-    /**
-     * Subscribe for incoming (late) submissions to show a message if the student submitted after the due date.
-     */
-    subscribeForNewSubmissions() {
-        this.studentParticipations.forEach((participation) => {
-            this.submissionSubscription = this.submissionService
-                // eslint-disable-next-line @typescript-eslint/no-non-null-asserted-optional-chain
-                .getLatestPendingSubmissionByParticipationId(participation!.id!, this.exercise?.id!, true)
-                .subscribe(({ submission }) => {
-                    // Notify about received late submission
-                    if (submission && !participation.testRun && getExerciseDueDate(this.exercise!, participation)?.isBefore(submission.submissionDate)) {
-                        this.alertService.success('artemisApp.exercise.lateSubmissionReceived');
```

Comment:
TODO: Remove this translation string from the translation JSON files (`lateSubmissionReceived` is now unused as far as I see).